app/models/crop.rb

```diff
@@ -167,7 +167,7 @@ def popular_plant_parts
     harvests.each do |h|
       popular_plant_parts[h.plant_part] += 1 if h.plant_part
     end
-    popular_plant_parts
+    popular_plant_parts.sort_by { |_s, freq| freq }.reverse
```

Comment:
I think this is is what upset the tests; not really sure if they are stale expectations or legit regression